lib/logstash/event.rb

```diff
@@ -16,6 +16,13 @@
 class LogStash::ShutdownEvent; end
 class LogStash::FlushEvent; end
 
+module LogStash
```

Comment:
I'd add an extra module level here to add a "subject" example:

```ruby
module LogStash::Signal
 FLUSH = LogStash::FlushEvent.new
 SHUTDOWN = LogStash::ShutdownEvent.new
end
```

Then:

`@input_to_filter.push(LogStash::Signal::SHUTDOWN)`

or

`@input_to_filter.push(Signal::SHUTDOWN)` since we're already inside the namespace.